scripts/data_collector/base.py

```diff
@@ -289,7 +289,18 @@ def __init__(
 
     def _executor(self, file_path: Path):
         file_path = Path(file_path)
-        df = pd.read_csv(file_path)
+
+        default_na = pd._libs.parsers.STR_NA_VALUES
```

Comment:
It would be great if we could add more context and comments here.
It is hard to understand the code without an explanation of motivation.

Reply:
Thank you for review! I added explanation comments:)